frontend/package.json

```diff
@@ -7,6 +7,9 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.4",
     "@testing-library/user-event": "^13.5.0",
+    "axios": "^0.27.2",
+    "init": "^0.1.2",
+    "moment": "^2.29.3",
```

Comment:
1. no one uses moment on the project so we don't need it here
2. never ever moment because it's deprecated: https://momentjs.com/docs/#/-project-status/

```suggestion
```